articles/transfer-learning-with-deep-learning/index.md

```diff
@@ -0,0 +1,121 @@
+
+### Useful Terms
+
+**Pre-trained models** – models trained on a sizeable benchmark dataset to solve a problem similar to an already solved problem.
+
+**Marginal probability** – The probability of an event in the presence of all outcomes of another random variable.
+
+**Feature** – an individual measurable property that is being observed. It is an attribute shared by all the independent units on that analysis is to be done.
+
+**Loss function** – component of a neural network responsible for calculating loss. Loss refers to the prediction error of a neural network.
+
+### Transfer Learning and Deep Learning
+
+In my previous transfer learning [article](https://www.section.io/engineering-education/basics-of-transfer-learning/), we defined transfer learning as a machine learning method where a model built for a specific task is reused as a starting point for a model on another task. It is basically re-using pre-trained models on new problems.
+
+Deep learning models have been applied to solve many problems that would have seemed impossible to solve a few years ago. Nonetheless, these models carry a few challenges with them. Firstly, they need very large amounts of data. They also need a lot of computing resources. Furthermore, they need a lot of time and energy to train.
+
+Deep neural networks have millions of weights that link many layers of neurons together. The weights are applied to inputs and adjusted during the training phase. The result is fed forward to provide an output. As we mentioned, this process is time-consuming and energy-intensive. This provides transfer learning a huge opportunity to optimize deep learning processes. Transfer learning allows us to use deep learning models that are pre-trained on a problem to be used on a new but related problem. This saves a lot of time and computational energy.
+
+#### Transfer Learning vs Traditional Machine Learning
+
+To understand transfer learning, it is important to compare it to a traditional machine learning approach. A key difference between the two is isolation. Traditional machine learning can be described as isolated. First things first, it focuses on specific tasks. It also occurs on specific datasets. Another characteristic of the traditional approach is the training of isolated models for very specific tasks. All this means that there is no knowledge that is either retained or accumulated. The learning process takes place without any relationship with past knowledge in different tasks.
+
+However, transfer learning takes the contrary approach to the isolated nature of traditional machine learning approaches. The basis of this approach is the transferability of prior knowledge to tackle new tasks. There is no isolation. It is possible to leverage knowledge from pre-trained models to train new models. Here, learning of a new task is dependent on tasks that were learned previously. The result of such is a faster and more efficient learning process.
```

Comment:
What do you mean by-> There is no isolation? Some more emphasis on that would be helpful.